update-scripts/nvim-lspconfig/locate-lsp-packages.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
+# This script requires nix-locate
```

Comment:
should this be a nix shebang with nix-locate included?

Reply:
I'm not sure, I think nix-locate needs a nix-index database to be installzd

Reply:
Oh, that sounds right... I think that's why I ended up adding nix-index-database to my flake.

Reply:
I could add a dependency on https://github.com/nix-community/nix-index-database for this, but it's maybe a bit heavy?

Reply:
Yeah, I don't know if it's worth an extra input.